### ISSUE TITLE
Custom builds

### ISSUE THREAD

**Issue by kmturley** [chrisdickinson/plate#13]
Plate is awesome and easily the fastest django templating library I've seen:
http://jsperf.com/jinja-client-side-libraries/2

However it is larger than the other templating libraries at 66KB

Is there a custom build which includes just the basic features?

**Comment by chrisdickinson** [comment]
I'll look into this. Are you thinking you'd like to selectively include / exclude certain tags and filters?

Thanks for the kind words!

**Comment by kmturley** [comment]
Yeah, some way of maybe excluding non core functionality so we can make a smaller build?

e.g. in jquery UI you can see that not all features are required for it to work
http://jqueryui.com/download/
